sdcflows/workflows/base.py

```diff
@@ -52,12 +52,14 @@
     'epi': 0,
     'fieldmap': 1,
     'phasediff': 2,
-    'syn': 3
+    'phase1': 3,
```

Comment:
Could you keep this change and send the rest of the changes as a separate PR? That will make it very easy to merge this very much necessary renaming of `bold_*` inputs/outputs.